plugins/Marketplace/templates/plugin-list.twig

```diff
@@ -2,151 +2,116 @@
-                                {{ plugin.description }}
-                                <a class="more" href="#" vue-directive="CorePluginsAdmin.PluginName"  vue-directive-value="{{ { pluginName: plugin.name }|json_encode }}" title="{{ 'General_MoreDetails'|translate }}">
-                                    &rsaquo; {{ 'General_MoreLowerCase'|translate }}</a>
-                            </p>
-
-                            {% if showThemes %}
-                                {# Screenshot for themes #}
-                                <a class="more" href="#" vue-directive="CorePluginsAdmin.PluginName"  vue-directive-value="{{ { pluginName: plugin.name }|json_encode }}">
-                                    <img title="{{ 'General_MoreDetails'|translate }}"
-                                         class="preview" src="{{ plugin.screenshots|first }}?w=250&h=150"/></a>
-                            {% endif %}
-
-                            <ul class="metadata">
-                                {% if plugin.isBundle is not defined or not plugin.isBundle %}
-                                    <li>
-                                        {% if plugin.latestVersion %}
-                                            {{ 'CorePluginsAdmin_Version'|translate }}: {{ plugin.latestVersion }}
-                                        {% endif %}
-
-                                        {% if plugin.canBeUpdated %}
-                                            <a class="update-available"
-                                                {% if plugin.changelog is defined and plugin.changelog and plugin.changelog.url is defined and plugin.changelog.url %}
-                                                    target="_blank" href="{{ plugin.changelog.url }}"
+                <div class="card-holder">
+                    {% embed 'contentBlock.twig' with {'title': ''} %}
+                        {% block content %}
+                            {% import '@Marketplace/macros.twig' as marketplaceMacro %}
+                            {% import '@CorePluginsAdmin/macros.twig' as pluginsMacro %}
+
+                                <img src="{{ plugin.previewImage }}" alt="" class="preview-image">
+                                <div class="content-container">
+                                    <div class="card-content-top">
+                                        {{ marketplaceMacro.matomoPluginBadge(plugin.owner, 'matomo-badge-top') }}
+                                        <div class="price">
+                                            {% if plugin.canBePurchased and plugin.priceFrom %}
+                                                {{ 'Marketplace_PriceFromPerPeriod'|translate(plugin.priceFrom.prettyPrice, plugin.priceFrom.period) }}
+                                            {% else %}
+                                                {{ 'Marketplace_Free'|translate }}
+                                            {% endif %}
```

Comment:
I think we should add an additional case here. If you have already added a license key and a premium plugin is included in that license, but isn't installed, the marketplace would now show `Free`. That is not fully correct. Maybe it would be better to show something like `purchased`, `already purchased`, `included in license` or similar.

Reply:
Yeah, I see what you mean. That would be a functional change as we don't have that covered with the old cards either, but I guess it makes sense. It's "free to install", but may not be covering the fact it's paid and purchased. I'll think about it and talk to Javi what we think is the best wording here.

Reply:
Discussed with the designer and we will always show the price when the plugin is paid, even when it's installable or already installed. It's an indication of price, not of whether it's been purchased or can be purchased or is for $0.